frontend/src/components/pages/profile/components/posts/Posts.tsx

```diff
@@ -10,7 +10,7 @@ import './posts.scss';
 const Posts: React.FC<IPostsProps> = ({ posts }) => {
   const sortedPosts = useMemo(() => {
     return posts.sort((a: any, b: any) => {
-      return (new Date(b.date).getTime() - new Date(a.date).getTime())
+      return new Date(b.date).getTime() - new Date(a.date).getTime();
```

Comment:
can be inline
return posts.sort((a: any, b: any) => new Date(b.date).getTime() - new Date(a.date).getTime()

DO NOT use any,  post has certain type -use it
Why cant we compare by post.date directly? 
Do we really need to sort? Posts should be saved in order

Reply:
I will be fixed by @OrestHaponov